clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3495,6 +3496,19 @@ as in the examples below:
        /* kernel code */
   });
 
+  class Functor
+  {
+      void operator()(item<1> item) [[intel::reqd_sub_group_size(16)]]
```

Comment:
This example isn't covered by a test.

Reply:
I mean that is exactly a 'right side' spelling. AFAIK clang isn't support it, considering an attribute to be a function type attribute, instead of function declaration, so I'm not expecting it to work 'out of the box'. Hence I'm not asking to add a test/or to add the appropriate code to support this spelling (as well as template instantiation) in a scope of this patch, but please correct the doc appropriately.

Reply:
@MrSidims .  Thanks for the review and explanation. I have corrected Doc since this PR does not handle "right side" attribute spelling and we will add this support in a separate PR.